bitrise.yml

```diff
@@ -24,7 +24,6 @@ workflows:
   test:
     steps:
     - go-list:
-    - golint:
```

Comment:
golint is already deprecated and doesn't exist in automatically generated `bitrise.yml` anymore.